indexing-service/src/main/java/io/druid/indexing/common/stats/TaskRealtimeMetricsMonitor.java

```diff
@@ -0,0 +1,122 @@
+/*
+ * Licensed to Metamarkets Group Inc. (Metamarkets) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. Metamarkets licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.druid.indexing.common.stats;
+
+import com.google.common.collect.ImmutableMap;
+import io.druid.java.util.emitter.EmittingLogger;
+import io.druid.java.util.emitter.service.ServiceEmitter;
+import io.druid.java.util.emitter.service.ServiceMetricEvent;
+import io.druid.java.util.metrics.AbstractMonitor;
+import io.druid.java.util.metrics.MonitorUtils;
+import io.druid.query.DruidMetrics;
+import io.druid.segment.realtime.FireDepartment;
+import io.druid.segment.realtime.FireDepartmentMetrics;
+
+import java.util.Map;
+
+/**
+ * Replaces the old RealtimeMetricsMonitor for indexing tasks that use a single FireDepartment, with changes to
```

Comment:
Would you add a comment to `RealtimeMetricsMonitor` as well that `RealtimeMetricsMonitor` is an old one and `TaskRealtimeMetricsMonitor` is recommended to use instead?

Reply:
Added a comment to RealtimeMetricsMonitor